duckduckgo/Android
# Round browser menu ripple corners

Task/Issue URL: https://app.asana.com/0/414730916066338/1201628734048281/f

### Description
Adds rounded corners to the back, refresh and settings ripples in the browser menu.

### Steps to test this PR

1. Open the browser overflow menu.
2. Verify that the back, forward and settings ripples are correctly rounded.

### UI changes
| Before  | After |
| ------ | ----- |
![back_before](https://user-images.githubusercontent.com/3471025/148560822-20ba9903-fa93-493a-b68a-e5fef9febe01.png)|![Screenshot_1641565249](https://user-images.githubusercontent.com/3471025/148560914-45e48dd4-932c-4d32-a90d-2c94388d7ed7.png)|
![refresh_before](https://user-images.githubusercontent.com/3471025/148561065-ecf3bfab-d6ad-40f5-931e-8ba839506ef3.png)|![Screenshot_1641565264](https://user-images.githubusercontent.com/3471025/148561108-30f8c134-9ea6-4a37-b022-8e49ae0d6847.png)|
![settings_before](https://user-images.githubusercontent.com/3471025/148561262-e98b4c4d-634b-4da9-a29a-1bc2fc640160.png)|![Screenshot_1641565190](https://user-images.githubusercontent.com/3471025/148561208-4f05b031-68f0-466f-9626-f28bd6d451ae.png)|
